package.json

```diff
@@ -56,7 +55,7 @@
   "devDependencies": {
     "browser-sync": "^2.26.7",
     "dotenv": "^8.1.0",
-    "eslint": "^6.5.1",
+    "eslint": "^5.16.0",
```

Comment:
Why downgrade?

Reply:
@pheeria oh, it was due to problems with `prettier-eslint`, but I found workaround to not use it and forgot to revert eslint version. Will do, thanks!